datacenter/ucp/2.0/guides/configuration/integrate-with-ldap.md

```diff
@@ -8,6 +8,9 @@ title: Integrate with LDAP
 Docker UCP integrates with LDAP services, so that you can manage users from a
 single place.
 
+**Note:** When you switch from the built-in authentication to using LDAP
```

Comment:
Just a little bit of rewording:

s/from the built-in/from using built-in

s/manually created/local

Also, try to wrap your lines around 80 characters so that diffs are easier to review later on. Thanks!